addStyles.js

```diff
@@ -95,11 +98,15 @@ function listToStyles(list) {
 	return styles;
 }
 
-function createStyleElement() {
+function createStyleElement(options) {
 	var styleElement = document.createElement("style");
 	var head = getHeadElement();
 	styleElement.type = "text/css";
-	head.appendChild(styleElement);
+	if (options.insertAt === "top") {
+		head.insertBefore(styleElement, head.firstChild);
+	} else if (options.insertAt === "bottom") {
+		head.appendChild(styleElement);
+	}
```

Comment:
What do you think of this for the error text: "Invalid value for parameter 'insertAt'. Must be 'top' or 'bottom'."